RelVal/interface/EnumerateBranches.h

```diff
@@ -1663,6 +1663,7 @@ template <>
 struct plotter <124> {
   constexpr static const char* name = "muons/size";
   std::vector<float> operator () (panda::Event& event) {
+    event.muons.sort(panda::Particle::PtGreater);
```

Comment:
I was thinking more along the lines that we could assert that the relevant output vector is already correctly sorted. So we'd have something like `assert(is_sorted(output))` in the struct with name `"muons/pt"`.

Reply:
Though we might want to be a little less aggressive than a literal `assert`, and change the exit code accordingly.